cli/uninstall/uninstall.go

```diff
@@ -82,3 +83,32 @@ func UninstallProgram(program string, binDst string, headerDst string,
 	log.Infof("%s is uninstalled.", program)
 	return err
 }
+
+// GetList generates a list of options to uninstall.
```

Comment:
```suggestion
// GetList generates a list of programs to uninstall.
```

Reply:
in the understanding of the shell, where it is passed is an option